x-pack/plugins/beats_management/public/pages/invalid_license.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import * as React from 'react';
+import { NoDataLayout } from '../components/layouts/no_data';
+
+export const InvalidLicensePage: React.SFC<any> = () => (
+  <NoDataLayout title="Invalid License" actionSection={[]}>
+    <p>
+      Your current license does not support Beats Central Management, or is expired. If you had
```

Comment:
The first sentence should say:

```
Your current license does not support Beats central management, or it's expired. 
```

I'm not sure about the second sentence because I'm not sure what you mean. do you mean that if the license expires, enrolled Beats will continue to work? If so, maybe say something like:

```
Beats that were enrolled before the license expired will continue to work.
```

Would really like @gchaps to weigh in here. I think this is a little wordy, but moves us in the right direction.